code-of-conduct.md

```diff
@@ -41,7 +43,11 @@ Examples of unacceptable behavior by participants include:
   asked
 * Discussion or promotion of activities or projects that intend or pose a risk of 
   significant harm
-* Trolling, insulting/derogatory comments, and personal or political attacks
+* Any discussion regarding religion and related topics
+* Discussion of politics and related topics, unless otherwise specific to 
+  electronics or technology
```

Comment:
Another thing I worry about here is that some groups have been heavily politicized and with how this is currently worded it can come off as unwelcoming to those groups - in a lot of cases "no politics" also means "no politically controversial people".

Reply:
@theacodes Do you have a suggestion for a more welcoming wording that still allows us to moderate inappropriate political discussions?

Reply:
@tannewt I can see it being its own list. It may, at that point, though, be worth coming up with a more comprehensive list.

Reply:
Maybe something along the lines of "no partisan politics" could be good

Reply:
I don't have a good solution, but some thoughts. As much as I was pro "Ban talk of politics and religion" it actually makes things far less welcoming, as @theacodes points out. For a not insignificant number of people, simply existing is politicized by others, and it shouldn't be on them to figure out how it goes for our space.

Instead of saying these topics are *prohibited* we can say something along the lines of "These topics can easily get out of hand and will be monitored closesly. The following specific subsections are prohibited"

Thus, we can prohibit things like knocking on religions or specific politicians, attempting to convert people to your religion/political party, saying which candidates are better to vote for, etc.

A full ban of Politics and Religion can be met with "Oh, I'm not going to post this neat story about a bunch of Sihkhs that pulled someone out of a river by tying their headwear into a rope because it's religious" or even "I'm not going to talk about my day spent planting trees because it was done as part of a political party". And it'll echo into related stuff like show and tell projects - Someone might not be willing to show a project that shows a daily text from the Quran because it might "provoke" a religious discussion and they don't want to be blamed for taht even though they're *technically* correct.

We don't want people to go say "Oh, hey, vote for X!" or "That religion's terrible!", no. But that's generally covered by other CoC rules.

I think we should make people more aware that these subjects can end up terribly, but honestly, thinking about it? I don't think we should fully ban them.

Reply:
Thank you all for your input. I have attempted to adjust the wording in that section to address provided feedback. Please let me know what you think.